ssc/cmod_pvsamv1.cpp

```diff
@@ -2013,6 +2013,7 @@ void cm_pvsamv1::exec( ) throw (general_error)
 						//DC batteries not allowed with multiple MPPT, so can just use MPPT 1's voltage
 						sharedInverter->calculateACPower(dcpwr_kw, PVSystem->p_mpptVoltage[0][idx], 0.0);
 						cliploss = sharedInverter->powerClipLoss_kW;
+                        PVSystem->p_systemACPower[idx] = sharedInverter->powerAC_kW;
```

Comment:
This code is only executed if there is not `p_pv_dc_clipping_forecast`. So if a clipping forecast is provided, what does the dispatch expect? Would a vector of zeroes make sense there or not?

Reply:
Oh, whoops, thanks for spotting this. A vector of zeroes does not make sense. It looks like some of the code for allowing for a custom PV forecast got stomped on (for example `p_pv_dc_forecast` was not passed into the battery), so I'll move things around a bit to re-enable that feature.